crates/astria-sequencer-relayer/tests/blackbox/helper.rs

```diff
@@ -20,17 +35,23 @@ use serde_json::json;
 use tempfile::NamedTempFile;
 use tendermint_config::PrivValidatorKey;
 use tendermint_rpc::{
-    endpoint,
     response::Wrapper,
     Id,
 };
 use tokio::{
+    net::TcpListener,
     sync::{
         mpsc,
         oneshot,
+        Mutex,
```

Comment:
Side comment: tokio recommends to prefer `std::sync::Mutex` over `tokio::sync::Mutex`. `tokio::sync::Mutex` is only needed if you plan to hold locks across await-points (which is tricky in general). In all other cases a `std::sync::Mutex` is better as one doesn't pay for the async-specific bits.